test/data/cryptoCurrencies.js

```diff
@@ -108,9 +108,9 @@ describe('the crypto currencies data module', () => {
       });
     });
 
-    it('that requires the canShapeShiftInto to be a boolean', () => {
```

Comment:
Would you mind adding a similar test for `canShapeShiftIntoPurchase` to ensure it's being provided as a boolean?

Reply:
Updated.